src/EnergyPlus/ExhaustAirSystemManager.cc

```diff
@@ -580,6 +571,7 @@ namespace ExhaustAirSystemManager {
                 thisExhCtrl.ExhaustFlowFractionScheduleNum = exhaustFlowFractionScheduleNum;
 
                 std::string supplyNodeOrNodelistName = ip->getAlphaFieldValue(objectFields, objectSchemaProps, "supply_node_or_nodelist_name");
+                thisExhCtrl.SupplyNodeOrNodelistName = supplyNodeOrNodelistName;
```

Comment:
You may be doing an extra deep copy by creating a temporary variable here.

Reply:
Thanks! Removed the temp variable and now it uses the struct member directly.